# model stands
# pcd_file_path: /home/iei/soeren_ws/src/perception/data/stands_model_pc.pcd
pcd_file_path: /home/donglzh/new_workspace/src/perception/data/stands_model_pc.pcd

